website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -672,6 +672,16 @@ A `windows_profile` block supports the following:
 
 * `license` - (Optional) Specifies the type of on-premise license which should be used for Node Pool Windows Virtual Machine. At this time the only possible value is `Windows_Server`.
 
+* `gmsa`- (Optional) A `gmsa` block as defined below
+
+---
+
+A `gmsa` block supports the following:
+
+* `dns_server` - (Optional) Specifies the DNS server for Windows gMSA. Set it to empty if you have configured the DNS server in the vnet which is used to create the managed cluster.
```

Comment:
Can you please confirm with the service team whether setting `dns_server` and `root_domain` to empty is a valid configuration for the cluster? This isn't mentioned in the documentation anywhere?